pkg/internal/checkup/checkup.go

```diff
@@ -180,6 +187,36 @@ func (c *Checkup) Run(ctx context.Context) error {
 	return nil
 }
 
+func (c *Checkup) checkVersions(ctx context.Context, errStr *string) error {
+	log.Print("checkVersions")
+
+	ver, err := c.client.GetClusterVersion(ctx, "version")
+	if err != nil {
+		return err
+	}
+	ocpVersion := ""
+	for _, update := range ver.Status.History {
+		if update.State == configv1.CompletedUpdate {
```

Comment:
and there is always only on update that has that state? or if not are they sorted by old->new? cause if not you will not get the last version necessarily. Just making sure :)

Reply:
See doc: `The newest update is first in the list and it is ordered by recency.`

Reply:
missed the `break` :)